ci/run_integration_test.sh

```diff
@@ -1,5 +1,17 @@
 #! /bin/bash -e
 
+aio_max_nr_recommended_value=1048576
+aio_max_nr=$(cat /proc/sys/fs/aio-max-nr)
+echo "The current aio-max-nr value is $aio_max_nr"
+if (( aio_max_nr ==  aio_max_nr_recommended_value  )); then
+   sudo sh -c  "echo 'fs.aio-max-nr = $aio_max_nr_recommended_value' >> /etc/sysctl.conf"
```

Comment:
have you run this in github actions yet ?

Reply:
No, but I think to remove it